spec/services/fetch_coronavirus_statistics_service_spec.rb

```diff
@@ -0,0 +1,124 @@
+RSpec.describe FetchCoronavirusStatisticsService do
+  describe ".call" do
+    it "returns a Statistics object" do
+      body = {
+        data: [
+          { "date" => "2021-03-18", "cumulativeVaccinations" => nil, "hospitalAdmissions" => nil, "newPositiveTests" => 6303 },
+          { "date" => "2021-03-17", "cumulativeVaccinations" => 25_735_472, "hospitalAdmissions" => nil, "newPositiveests" => 5758 },
+          { "date" => "2021-03-16", "cumulativeVaccinations" => 25_273_226, "hospitalAdmissions" => nil, "newPositiveests" => 5294 },
+          { "date" => "2021-03-15", "cumulativeVaccinations" => 24_839_906, "hospitalAdmissions" => nil, "newPositiveests" => 5089 },
+          { "date" => "2021-03-14", "cumulativeVaccinations" => 24_453_221, "hospitalAdmissions" => 426, "newPositiveests" => 4618 },
+          { "date" => "2021-03-13", "cumulativeVaccinations" => 24_196_211, "hospitalAdmissions" => 460, "newPositiveests" => 5534 },
+        ],
+      }
```

Comment:
This is great! Thanks for clarifying what we expect data from coronavirus.data.gov.uk to look like ❤️